samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -297,3 +300,104 @@ SnapToCenterVirtualGridList.storyName = 'Snap to center VirtualGridList';
 SnapToCenterVirtualGridList.parameters = {
 	propTables: [Config]
 };
+
+class VirtualGridListInScroller extends Component {
+	renderItem = ({index, ...rest}) => {
+		const
+			color = Math.floor((Math.random() * (0x1000000 - 0x101010)) + 0x101010).toString(16),
+			source = {
+				'hd': `http://placehold.it/200x200/${color}/ffffff&text=Image ${index}`,
+				'fhd': `http://placehold.it/300x300/${color}/ffffff&text=Image ${index}`,
+				'uhd': `http://placehold.it/600x600/${color}/ffffff&text=Image ${index}`
+			},
+			text = `Item ${index}`;
+
+		return (
+			<ImageItem {...rest} label={text} onClick={this.props.onNext} src={source} />
```

Comment:
One more thing..
We have `childProps` to pass props like `onClick` since it's using internal props.

Reply:
It's applied.